build.sbt

```diff
@@ -8,20 +8,18 @@ autoScalaLibrary := false
 // Remove Scala version in output paths and artifacts
 crossPaths := false
 
-javacOptions ++= Seq("-source", "1.6", "-target", "1.6", "-Xlint:deprecation")
-
-javacOptions in doc := Seq("-source", "1.6")
```

Comment:
Hi, why do we remove the options `"-source", "1.6", "-target", "1.6"`?

Reply:
In the "Compile Your Application if Needed" section of the Migration Guide to JDK11, it is stated that targeting 1.6 is deprecated in JDK11 and that the specification of target and source will change in the future as supported versions change.

However, it may be more appropriate to include the release option instead.

In any case, it would be better not to target Java 6 at this time.

https://docs.oracle.com/en/java/javase/11/migrate/index.html

Reply:
The purpose of setting that option is to have a stable target version.

If we don't have that option, imagine this scenario:
* I build the library with Java 11 and publish the library to Maven.
* You use Java 8, you download the library, and you cannot use it, because the class files are not compatible with Java 8.

So removing the option will cause class file compatibility problem.

If you don't like setting `1.6`, please modify the pull request to at least set the option to `1.8`.